frontend/src/pages/ProgressionChecker/CoursesModal/CoursesModal.tsx

```diff
@@ -43,15 +44,10 @@ const CoursesModal = ({
       </S.SortBtnWrapper>
       <S.CourseList>
         {courses.sort(applySortFn).map((course) => (
-          <CourseBadge
+          <CourseCard
```

Comment:
Know it was in the ticket description, but revert this once it has been merged to dev